src/schema/v2/me/myCollectionCreateArtworkMutation.ts

```diff
@@ -154,16 +153,29 @@ export const myCollectionCreateArtworkMutation = mutationWithClientMutationId<
       createArtworkLoader,
       createArtworkImageLoader,
       createArtworkEditionSetLoader,
+      createArtistLoader,
     }
   ) => {
     if (
       !createArtworkLoader ||
       !createArtworkImageLoader ||
-      !createArtworkEditionSetLoader
+      !createArtworkEditionSetLoader ||
+      !createArtistLoader
     ) {
       return new Error("You need to be signed in to perform this action")
     }
 
+    if (!artistIds?.length && !artists?.length) {
+      return new Error("You need to provide either artist IDs or artists")
+    }
+
+    // Create artists if `artist` is provided in the input fields
+    if (artists?.length) {
+      const newArtistIDs = await createArtists(artists, createArtistLoader)
+
+      artistIds = [...(artistIds || []), ...newArtistIDs]
```

Comment:
It's not really necessary at the moment, but I thought it would be useful because in the future we might want to link several artists to one artwork and mix Artsy's artists and personal/custom artists.